### DISCUSSION TITLE
words get cut off at the end of a chunk.

### DISCUSSION THREAD

**Discussion by Bonk1971** [manmay-nakhashi/tortoise-tts-fastest#3] [General]
I am not sure what setting I need to adjust to get the words to finish at the end of a segment.  I am processing fairly large texts and I am struggling to get the words to finish consistently at the end of the segment being processed.  What do I need to adjust?

**Comment by manmay-nakhashi** [comment]
@Bonk1971 Increase mel_mel_length.

**Reply by Bonk1971** [reply]
are you referring to max_mel_length?

**Reply by manmay-nakhashi** [reply]
Yes , or may be your text is too long, in that case you have to split your sentences.
